### ISSUE TITLE
[Announcement] GitHub Issues Will Be Closed by 30 Nov

### ISSUE THREAD

**Issue by danqiye1** [nus-cs2030/2223-s1#307]
Hi students of CS2030,

The Github issues page and wiki will be closed from 30 Nov onwards. Please make your final contributions to the issues and the wiki before that date.

**Comment by aerodromic** [comment]
Can we create new pages in the wiki?

**Comment by danqiye1** [comment]
Nope, by then I will close the wiki as well. If you want to do it. Today is the last day.
